Import notes.md into readme.md (issue #95)

SimonGreenhill opened the pull request.

xrotwang commented:
things are moving quickly

LinguList commented:
Okay, cool! I'd suggest we sign the notes with our github username, or something similar. In this way, we have also a "responsible editor" for a given dataset, and people who have questions might directly turn to the one that has curated a given dataset?

In general, I think this is how it should be in the future: who wishes to add data approaches one of us (and future collaborators to come), asking to add a dataset, if they're not capable of doing it themselves. And every dataset should normally have one responsible lexibankler to take care of adding it, while the others only review the code.

SimonGreenhill commented:
I've added a comment into [workflow.md](https://github.com/glottobank/lexibank-data/blob/master/workflow.md) to this effect. 

Question: do we want to sign these, or include the output of git log as a `history` section in each README.md file? e.g. `git log mixezoquean` gives:

```
commit 2eb6312ed2b436aabde1946870313cad9f587846
Author: LinguList <mattis.list@uni-marburg.de>
Date:   Sat Sep 24 19:30:25 2016 +0200

    updated, illustrating how preparsing can be used in clean_string

commit 9c857fadcf09cf6d7dbc111d8009df28f440fc8b
Author: LinguList <mattis.list@uni-marburg.de>
Date:   Sat Sep 24 19:07:23 2016 +0200

    adjusting small error

commit bb208a2f50e6ee29bb3957c1b251e3725a22b12e
Author: LinguList <mattis.list@uni-marburg.de>
Date:   Sat Sep 24 18:59:07 2016 +0200

    fixes #84

commit 72b7a39d07326cd7d7bef2b953f850a7576347f6
Author: LinguList <mattis.list@uni-marburg.de>
Date:   Sat Sep 24 18:56:58 2016 +0200

    updated data

commit d34ffce0431ae2d972c945b6bb61b8f323e7e957
Author: LinguList <mattis.list@uni-marburg.de>
Date:   Sat Sep 24 18:36:28 2016 +0200

    updated data
```
